src/Humanizer/EnumHumanizeExtensions.cs

```diff
@@ -49,7 +50,17 @@ private static string GetCustomDescription(MemberInfo memberInfo)
                 {
                     var method = attrType.GetMethod(DisplayAttributeGetDescriptionMethodName);
                     if (method != null)
-                        return method.Invoke(attr, new object[0]).ToString();
+                    {
+                        var obj = method.Invoke(attr, new object[0]);
+                        if (obj != null) return obj.ToString();
+                    }
+                    var displayNameMethod = attrType.GetMethod(DisplayAttributeGetNameMethodName);
+                    if (displayNameMethod != null)
+                    {
+                        var obj = displayNameMethod.Invoke(attr, new object[0]);
+                        if (obj != null) return obj.ToString();
+                    }
+                        
                 }
                 var descriptionProperty =
```

Comment:
It would be great if you could also prevent the `NullReferenceException` when the value of the custom description property is null.